packages/fondue/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -239,6 +268,15 @@ export const MultiSelect = ({
 
                             {activeItemKeys.length === 0 && placeholder && <Text color="weak">{placeholder}</Text>}
                         </div>
+                        {filter.enabled && (
+                            <input
+                                data-test-id="filter-input"
+                                ref={filterInputRef}
+                                className="tw-outline-none tw-bg-transparent tw-placeholder-black-60 tw-text-s"
```

Comment:
Alignment of the text input should follow the chips and wrap on a new row, but it's centered in a weird way right now: 
![Screenshot 2024-03-14 at 09 59 25](https://github.com/Frontify/fondue/assets/45793869/e200e3a2-fe39-4ecd-bb45-5ee4c30ed2eb)

Reply:
Fully agreed. Checked it out with our designer and changed the input to be rendered inline after the active items.